### PR TITLE
docs(documentation) transparent proxying section

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
Signed-off-by: Jakub Dyszkiewicz <jakub.dyszkiewicz@gmail.com>

### CODE REVIEW

File: docs/docs/0.7.1/documentation/networking.md [hunk 1]
```diff
@@ -188,3 +188,79 @@ Kuma DNS allocates a VIP for every Service withing a mesh. Then, it creates outb
...
+type: Dataplane
+...
+networking:
+  ...
+  outbound:
+  - port: 10000
+    tags:
+      kuma.io/service: backend
+```
+This approach has the disadvantage that you need to reconfigure the service to use `http://localhost:10000` when it wants to connect with service `backend`.
```
Comment (nickolaev) [inline docs/docs/0.7.1/documentation/networking.md:206]: Explain that this is what we use in Universal.

File: docs/docs/0.7.1/documentation/networking.md [hunk 2]
```diff
@@ -188,3 +188,79 @@ Kuma DNS allocates a VIP for every Service withing a mesh. Then, it creates outb
...
+
+The alternative approach is Transparent Proxying. With Transparent Proxying before we start a service, we apply [`iptables`](https://linux.die.net/man/8/iptables) that intercept all the traffic on VM/Pod and redirect it to Envoy.
```
Comment (nickolaev) [inline docs/docs/0.7.1/documentation/networking.md:208]: Do we have to rpovide some setting to DP to know it is in transparent proxy mode?

Comment (jakubdyszkiewicz) [reply docs/docs/0.7.1/documentation/networking.md:209]: nope

File: docs/docs/0.7.1/documentation/networking.md (hunk 1 above)
Comment (jakubdyszkiewicz) [reply docs/docs/0.7.1/documentation/networking.md]: added

**Pull request merged**